chapters/introduction.tex

```diff
@@ -69,10 +69,11 @@ \chapter{Introduction}
 
 \begin{figure}[htp]
 \centering
-  \includegraphics[width=\columnwidth]{images/assignment-schema.pdf}
   \caption[\ac{cr} assignment.]{\ac{cr} assignment. The router, which may be the
   \acs{ccb}, project leaders, or managers, must match \acp{cr} and developers in
   order to obtain the shortest fixing and highest quality.}
+  \includegraphics[width=\columnwidth]{images/assignment-schema.pdf}
+  \footnotesize{Source: Made by the author.}
```

Comment:
Forgot to add in the summary of pull request: Captions must be on the top of figures. Every figure must cite the source, even if made by the own author.